styles/push-pull-view.less

```diff
@@ -2,6 +2,11 @@
 
 // Used in the status-bar
 
+// click target should take up the width of the container
+.push-pull-target {
```

Comment:
This works for the height, but for the One UI themes, the `width` doesn't go all the way to the edge because the parent [github-PushPull](https://github.com/atom/github/blob/4519a1217313591aae9b672ee3451bebe6c7ed1a/lib/views/push-pull-view.js#L201-L203) element has some padding:

![image](https://user-images.githubusercontent.com/378023/41580952-d9df8702-73d7-11e8-8fcd-153ad4d4fa8b.png)

Hmm.. would it be possible to move the click target to that parent `github-PushPull` element? And remove the `<a class="push-pull-target">`?

If not or if there is a lot of refactoring needed, we could also do this:

1. Merge this PR as is. Fixes height.
2. Add some "negative margin hack" to the One UI themes.

Like:

```less
.push-pull-target {
  display: block;
  margin: 0 -0.75em;
  padding: 0 0.75em;
}
```

Reply:
thanks for the feedback! I can take a stab at removing the `pushPull` element.